### ISSUE TITLE
Skyrim, Get Missing download ID not working for three mods

### ISSUE THREAD

**Issue by h-lucz** [Nexus-Mods/NMM-Test-Builds#79]
Get Missing Download ID worked for Lightning during thunderstorms - CoT 3.1 patch - retrieve DL ID.
It does not work (missing mod name in information field) for all the three mods as displayed in attached image.

![skyrim get missing dl id not working](https://cloud.githubusercontent.com/assets/18660199/15223273/e268da8e-1873-11e6-864c-4a76aa837e46.png)

**Comment by KunoMochi** [comment]
I feel that this is probably related: https://github.com/Nexus-Mods/NMM-Test-Builds/issues/76
Run "Check for missing/outdated download IDs" and see if that helps.

**Comment by h-lucz** [comment]
The issue above has remained for the mod Blacksmith forge fix (no DL ID at all and the two SoS patches marked with "No matches on the Nexus server" in DL ID columnd and the Mod version marked with ?) with latest version 62.11. I tried "check for missing/outdated Download IDs" first and after that "Gets missing mod info". With try and error and copy and paste of mod name I could make "Gets missing mod info" run. For Blacksmith forge fix the version number was updated, but no DL ID.  A Downlaod ID has not been found. Maybe due to the authors archive that contains several esp files for Vanilla, DB and Legendary plus 4 corresponding language version (loose, not with a fomod installer).
